src/components/ModalWithForm.js

```diff
@@ -1,11 +1,16 @@
 import Modal from "./Modal.js";
 
 export default class ModalWithForm extends Modal {
-  constructor(modalSelector, handleFormSubmit) {
+  constructor({ modalSelector, handleFormSubmit }) {
     super({ modalSelector });
     this._modalForm = this._modalElement.querySelector(".modal__form");
     this._handleFormSubmit = handleFormSubmit;
     this._inputList = [...this._modalForm.querySelectorAll(".modal__input")];
+    this._saveButton = this._modalForm.querySelector(".modal__button");
+  }
+
+  confirmDelete(handleFormSubmit) {
```

Comment:
Please, rename `confirmDelete` in a neutral way because `ModalWithForm` is  a **universal** class for any modal with a form


`setSubmitHandler`